accumulate_test.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"github.com/cloudscale-ch/cloudscale-go-sdk/v2"
+	"testing"
+	"time"
+)
+
+func Test(t *testing.T) {
+	zone := "appuio-cloudscale-ch-lpg"
+	tenant := "inity"
+	namespace := "testnamespace"
+
+	// get the correct date for a data set that was created yesterday
+	location, err := time.LoadLocation("Europe/Zurich")
+	if err != nil {
+		t.Errorf("could not load location Europe/Zurich")
+	}
```

Comment:
We usually use the [testify](https://github.com/stretchr/testify/) library for asserting in tests. It's a set of additional assertions similar to Junit for Java.

Quick intro:
testify has 2 basic assertion packages: `assert` and `require`. They have both similar assertions (e.g. `NoError()`), but `require` aborts test execution if the assertion fails. Assertions in the `assert` package continue the test code.
This can be useful in case the following code after an assertion would fail in any case (e.g. Nil pointer errors), so use `require.xxx()` if following assertions would fail anyway.

When executing `go test` the error messages will be enriched.

```suggestion
	location, err := time.LoadLocation("Europe/Zurich")
	require.NoError(t, err, "could not load location Europe/Zurich")
```